README.md

```diff
@@ -273,6 +284,20 @@ script: matlab -batch "addpath('code'); results = runtests('IncludeSubfolders',
 ```
 <br>
 
+### GitLab CI/CD
+```yml
+stages:         
+  - matlab-test
+
+matlab-test:       
+  stage: matlab-test
+  script:
+    - matlab -batch "addpath('code'); results = runtests('IncludeSubfolders', true); assertSuccess(results);"
+  tags:
+    - matlab #Optional, use tags to select a specific runner from the list of all runners that are available for the project.
+```
```

Comment:
Need to update per changes to .gitlab.yml file.

Reply:
Do we need to delete the tags related line?